docs/COMPLETING.md

```diff
@@ -0,0 +1,192 @@
+
+const Routes = (
+    <Route path="/" component={App}>
+        <IndexRoute component={IndexContainer} />
+    </Route>
+);
+
+export default Routes;
+```
+
+---
+
+On the seventh line we define that the root URL matches our previously made `App`
+```typescript
+import * as React from 'react';
+import { Route } from 'react-router';
+import App from './App';
+// ...
+    <Route path="/" component={App}>
+        // ...
+    </Route>
+```
+using [`Route`](https://github.com/ReactTraining/react-router/blob/v3/docs/API.md#route) which takes as the first argument `path` the URL this `Route` should match to (*relative URL*) and as the second argument `component` the component to render.
+
+---
+
+On the eighth line we define our [`IndexRoute`](https://github.com/ReactTraining/react-router/blob/v3/docs/API.md#indexroute-1) for the root path to be `IndexContainer`
+```typescript
+import * as React from 'react';
+import { IndexRoute } from 'react-router';
+import IndexContainer from './index/IndexContainer';
+// ...
+        <IndexRoute component={IndexContainer} />
+```
+which is a special type of `Route` as it defines the `component` it received as the child for it's parent's route (*meaning that for the parent's path, this is the innermost component to show*). Notice that we are setting the `Container` as the `component` instead of the `View`. This is what allows us to connect **Redux** to our **React**-components in the next phase.
+
+### <a name="index">index</a>
+
+Finally we create a file `index.ts` inside `src`
+```typescript
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { Router, browserHistory } from 'react-router';
+import configureStore from './redux/store';
+import Routes from './modules/Routes';
+
+ReactDOM.render((
+    <Provider store={configureStore()}>
+        <Router history={browserHistory} routes={Routes} />
+    </Provider>
+    ), document.getElementById('app'),
+);
+```
+which is the entry file to our application that ties everything together.
+
+---
+
+On the eighth line
+```typescript
+import * as ReactDOM from 'react-dom';
+
+ReactDOM.render((
+    // ...
+    ), document.getElementById('app'),
+))
+```
+we [render](https://facebook.github.io/react/docs/react-dom.html#render) our **React**-application to the **DOM** inside a div with the id `app` (*we'll come back to this*).
+
+---
+
+On the ninth line
+```typescript
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import configureStore from './redux/store';
+// ...
+    <Provider store={configureStore()}>
+        // ...
+    </Provider>
+```
+which wraps our **React** application with **Redux** using [`Provider`](https://github.com/reactjs/react-redux/blob/master/docs/api.md#provider-store) from **react-redux**, which takes a single parameter `store`, for which we provide our store as we defined it in [Redux](/REDUX.md#store).
```

Comment:
Completing is one of the last sections in the documentation so it should've been defined already :)